service/resources/dispatch_email.py

```diff
@@ -0,0 +1,157 @@
+"""Welcome example module"""
+#pylint: disable=too-few-public-methods
+import sys
+import os
+import json
+import logging
+import datetime
+import pytz
+import falcon
+import sentry_sdk
+import jsend
+from ..modules.common import get_airtable, email, get_accela_link_by_id
+
+ERROR_GENERIC = "Bad Request"
+ERROR_401 = "Unauthorized"
+
+class Email():
+    """Email class"""
+    def on_post(self, req, resp):
+        #pylint: disable=no-self-use
+        """on post request
+        """
+        msg = ERROR_GENERIC
+
+        if req.content_length:
+            data = req.stream.read(sys.maxsize)
+            data_json = json.loads(data)
+
+            try:
+                #pylint: disable=line-too-long
+                if 'airtable_record_id' not in data_json or data_json['token'] != os.environ.get('EMAIL_TOKEN'):
+                    raise ValueError(ERROR_401)
+
+                # init airtable
+                airtable_id = data_json['airtable_record_id']
+
+                emails_sent = self.send_submission_email_by_airtable_id(airtable_id)
+                msg = emails_sent
```

Comment:
this seems extra.  why not just use emails_sent directly?  or set msg = self.send_sub...?

Reply:
Thanks. Updated in 653f710307747b27d7d20a96f3f1af779bf34714